src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderInput.tsx

```diff
@@ -318,31 +322,29 @@ export const LimitOrderInput = ({
         isCompact={isCompact}
         isError={false}
         isLoading={isLoading}
-        manualAddressEntryDescription={undefined}
         onRateClick={noop}
         quoteStatusTranslation={'trade.previewTrade'}
         rate={activeQuote?.rate}
-        receiveAddress={manualReceiveAddress ?? walletReceiveAddress}
-        recipientAddressDescription={undefined}
         sellAsset={sellAsset}
-        sellAssetAccountId={sellAssetAccountId}
-        shouldDisablePreviewButton={false}
-        shouldForceManualAddressEntry={false}
+        sellAssetAccountId={sellAccountId}
```

Comment:
nitpick: could unify naming here in props as well if we're going for this terminology change

Reply:
Ok I've unified as much as I can in an upcoming commit within the context of limit orders, but doing so completely is going to step out of scope for this PR:
<img src="https://github.com/user-attachments/assets/a4af3bde-ab10-4482-b325-7ae7818c272b" width="200">
<img src="https://github.com/user-attachments/assets/c34791d4-e313-460f-a73c-cc369ce99301" width="200">

Reply:
4c54e5482b82ab42b7167f03a4fe62eb92dfd8a5